plugins/faustwp/composer.json

```
@@ -23,6 +23,16 @@
     "yoast/phpunit-polyfills": "^1.0"
   },
   "scripts": {
+    "docker:build": "bash bin/run-docker.sh build",
+    "docker:serve": [
+      "@docker:stop",
+      "docker compose up -d"
```

Thoughts on updating this to use the full flag since `--volumes` is being used below.
```suggestion
      "docker compose up --detach"
```